src/FsAutoComplete.Core/Commands.fs

```diff
@@ -47,6 +47,7 @@ type Commands<'analyzer> (serialize : Serializer, backgroundServiceEnabled) =
     let state = State.Initial (checker.GetFSharpChecker())
     let fileParsed = Event<FSharpParseFileResults>()
     let fileChecked = Event<ParseAndCheckResults * string * int>()
+    let scriptFilePropjectOptions = Event<FSharpProjectOptions>()
```

Comment:
typo here: `scriptFilePropjectOptions` ==> `scriptFileProjectOptions`. I'd do a github suggestion but there are other places where this needs to be renamed as well.